src/lib/utilities/request-from-api.ts

```diff
@@ -47,6 +47,9 @@ export const requestFromAPI = async <T>(
     retryInterval = 5000,
   } = init;
 
+  const endpointParams = endpoint.split('?')[1];
+  endpoint = endpoint.split('?')[0];
+
   if (!endpoint.startsWith('/')) endpoint = '/' + endpoint;
   const nextPageToken = token ? { next_page_token: token } : {};
   const query = new URLSearchParams({
```

Comment:
This is _also_ the query—so, do we already have the query parameters stored? If so, we don't need to do it twice.